package.json

```diff
@@ -126,7 +126,7 @@
     "rimraf": "^3.0.2",
     "ts-jest": "^25.3.0",
     "typescript": "^3.9.5",
-    "webpack": "^4.42.0",
+    "webpack": "https://github.com/webpack/webpack",
```

Comment:
Leaving a comment here, this to be updated before we merge

Reply:
done

Reply:
nicenice